pages/test_analytics/ci_environments.md.erb

```diff
@@ -0,0 +1,108 @@
+Make sure to export your CI environment's variables and your Buildkite API token to your containerized builds and tests.
+
+For example, by default Docker does not receive the host's environment variables.
+To pass them through to the Docker container, use the `--env` option:
+
+```
+  docker run \
+    --env TEST_ANALYTICS_TOKEN \
+    --env BUILDKITE_BUILD_NUMBER \
+    --env BUILDKITE_JOB_ID \
+    --env BUILDKITE_BRANCH \
+    --env BUILDKITE_COMMIT \
+    --env BUILDKITE_MESSAGE \
+    --env BUILDKITE_BUILD_URL \
+    bundle exec rspec
+```
+
+Review the following sections for the environment variables expected by test collectors.
+
+## Buildkite
+
+During Buildkite pipeline runs, test collectors upload information from the following environment variables:
+
+| Environment variable     | Description                                   |
+| ------------------------ | --------------------------------------------- |
+| `BUILDKITE_BRANCH`       | the branch or reference for this build        |
+| `BUILDKITE_BUILD_ID`     | the UUID for the build                        |
+| `BUILDKITE_BUILD_NUMBER` | the build number                              |
+| `BUILDKITE_BUILD_URL`    | the URL for the build on Buildkite            |
+| `BUILDKITE_COMMIT`       | the commit hash for the head of the branch    |
+| `BUILDKITE_JOB_ID`       | the job UUID                                  |
+| `BUILDKITE_MESSAGE`      | the commit message for the head of the branch |
+
+## CircleCI
+
+During CircleCI workflow runs, test collectors upload information from the following environment variables:
+
+| Environment variable | Description                                |
+| -------------------- | ------------------------------------------ |
+| `CIRCLE_BRANCH`      | the branch or reference being built        |
+| `CIRCLE_BUILD_NUM`   | the number for the job                     |
+| `CIRCLE_BUILD_URL`   | the URL for the job on CircleCI            |
+| `CIRCLE_SHA1`        | the commit hash for the head of the branch |
+| `CIRCLE_WORKFLOW_ID` | the unique identifier for the workflow run |
+
+## GitHub Actions
+
+During GitHub Actions workflow runs, test collectors upload information from the following environment variables:
+
+| Environment variable | Description                                             |
+| -------------------- | ------------------------------------------------------- |
+| `GITHUB_ACTION`      | the name of the action running or its step ID           |
+| `GITHUB_REF`         | the ref (branch or tag) that triggered the workflow run |
+| `GITHUB_REPOSITORY`  | the repository owner and repository name                |
+| `GITHUB_RUN_ATTEMPT` | the numbered attempt of the workflow run                |
+| `GITHUB_RUN_ID`      | the unique number for the workflow run                  |
+| `GITHUB_RUN_NUMBER`  | the cumulative number of runs for the workflow          |
+
+## Other CI providers
```

Comment:
🚨  New section on manually providing environment variables! 🚨 

This was more interesting than I first imagined. Unlike the other sections, where we just need to list out the preexisting variables, here we need to tell folks what to set those variables _to_, which calls for examples. And then I realized that we'll do need to the correspondence to the API. And, well, this table got a little out of hand. There's probably a prettier way to show this stuff (perhaps Oly will have thoughts), but this is close to good enough for now, I think.